src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Models/AzureFileShareModels/AzureFileSharePolicy.cs

```diff
@@ -17,17 +17,8 @@ namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models
     /// <summary>
     /// Azure FileShare specific backup policy class.
     /// </summary>
-    public class AzureFileSharePolicy : PolicyBase
+    public class AzureFileSharePolicy : AzureWorkloadPolicy
```

Comment:
I think this should be renamed to AzurePolicy instead of AzureWorkloadPolicy, to maintain consistency with other similar objects.